torch/nn/parallel/distributed.py

```diff
@@ -36,8 +37,7 @@
 from torch._utils import _get_device_index
 
 from ..modules import Module
-from ._functions import _get_stream
-from .scatter_gather import gather, is_namedtuple, scatter_kwargs
+from .scatter_gather import gather, scatter_kwargs
```

Comment:
I may have missed it, but where is `gather()` used?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/nn/parallel/distributed.py#L1135, it is not used by DDP, but I guess it is a public interface, so we cannot easily remove it